src/utils/user_utils.js

```diff
@@ -21,16 +22,14 @@ export function displayUsername(
     user: UserProfile,
     teammateNameDisplay: Preferences.DISPLAY_PREFER_NICKNAME | Preferences.DISPLAY_PREFER_FULL_NAME
 ): string {
-    let name = '';
+    let name = localizeMessage('channel_loader.someone', 'Someone');
```

Comment:
In order to test this, we need to mock this import. Should we add a library for this?